src/Lykke.blue.Service.ReferralLinks/Startup.cs

```diff
@@ -91,6 +96,20 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
                     app.UseDeveloperExceptionPage();
                 }
 
+                app.UseLykkeMiddleware(Constants.ComponentName, ex =>
+                {
+                    Log.WriteFatalErrorAsync(Constants.ComponentName, "", ex);
+
+                    string errorMessage = "Technical problem";
+
+                    if (!String.IsNullOrWhiteSpace(ex.Message))
+                    {
+                        errorMessage = ex.Message;
```

Comment:
Why we provide internal error for clients?
here need send "Technical problem" and after find problim by logs

Reply:
Done.